kornia/losses/total_variation.py

```diff
@@ -1,19 +1,26 @@
-import torch
 import torch.nn as nn
 
+from kornia.core import Tensor
+from kornia.testing import KORNIA_CHECK, KORNIA_CHECK_SHAPE
 
-def total_variation(img: torch.Tensor) -> torch.Tensor:
+
+def total_variation(img: Tensor, reduction: str = "sum") -> Tensor:
```

Comment:
any strong opinion why `sum` by default ? i think we default to reduction `none` in some places. Just to standardize over the lib

Reply:
to be backward compatible

Reply:
fair enough